### PR TITLE
docs: polish get started

### PR DESCRIPTION
**Pull request by gao-sun**
- reorg the get started part for a better reading experience
- add Docker Compose and Docker installation

image

### CODE REVIEW

File: docs/docs/tutorials/get-started/README.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+Open the link and you will see a welcome page that briefly introduces Admin Console. Click "Create Account" to create an admin account.
+
+:::note
+For now, you cannot create multiple admin accounts via Admin Console, and only the first account will have admin access. If you want to create multiple admin accounts, use the Management API instead.
+:::
+
+:::caution
+Once you create the first account, the welcome page will be unavailable.
+:::
```
Comment (IceHe) [inline docs/docs/tutorials/get-started/README.mdx:93]: nit: maybe unnecessary? Seems that the welcome page should usually show up only once.

File: docs/docs/tutorials/get-started/README.mdx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+- [Node.js](https://nodejs.org/) `^16.13.0`
+- [PostgreSQL](https://postgresql.org/) `^14.0`
+
+Higher versions usually work but are not guaranteed.
+
+We recommend using a new empty database which is dedicated for Logto, while it's not a hard requirement.
+
+**Download and start**
+
+In your terminal:
```
Comment (IceHe) [inline docs/docs/tutorials/get-started/README.mdx:56]: nit:
```suggestion
Execute the following command in your terminal:
```

File: docs/docs/tutorials/get-started/README.mdx (hunk 1 above)
Comment (gao-sun) [reply docs/docs/tutorials/get-started/README.mdx:93]: there's a confusion that another admin can also use this page to create account

File: docs/docs/tutorials/get-started/README.mdx (hunk 2 above)
Comment (gao-sun) [reply docs/docs/tutorials/get-started/README.mdx:56]: well i think it's ok to stay with "in"

File: docs/docs/tutorials/get-started/README.mdx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+```bash
+node -e "$(printf "%s" "$(curl -fsSL https://raw.githubusercontent.com/logto-io/logto/HEAD/install.js)")"
+```
+
+The script will download Logto and create a directory `logto` in the location you ran it. After answering [several simple questions](../../references/core/configuration.md#questions), you will see the message like:
+
+</TabItem>
```
Comment (demonzoo) [inline docs/docs/tutorials/get-started/README.mdx:64]: Do we miss a screenshot after the `see the message like:`?

File: docs/docs/tutorials/get-started/README.mdx (hunk 1 above)
Comment (demonzoo) [reply docs/docs/tutorials/get-started/README.mdx:93]: unavailable is good to me

**Review by darcyYe — APPROVED** [review]

**Review by demonzoo — APPROVED** [review]

File: docs/docs/tutorials/get-started/README.mdx (hunk 3 above)
Comment (gao-sun) [reply docs/docs/tutorials/get-started/README.mdx:64]: No, see the following part. They share a common “message”

**Pull request merged**